airbyte-integrations/connectors/source-aws-cloudtrail/source_aws_cloudtrail/spec.json

```diff
@@ -14,20 +14,24 @@
     "properties": {
       "aws_key_id": {
         "type": "string",
-        "description": "Specifies an AWS access key associated with an IAM user or role.",
+	"title": "Key ID",
+        "description": "AWS CloudTrail Access Key ID. See the <a href=\"https://docs.airbyte.io/integrations/sources/aws-cloudtrail\">docs</a> for more information on how to obtain this key.",
         "airbyte_secret": true
       },
       "aws_secret_key": {
         "type": "string",
-        "description": "Specifies the secret key associated with the access key. This is essentially the 'password' for the access key.",
```

Comment:
Same as described in this comment: https://github.com/airbytehq/airbyte/pull/8434/files#r763024940